mantidproject/mantidimaging
# Show the name of the stack being compared as a title on compare window

Fixes #673 

To test:
- Load data with 2 or more stacks
- Open the Operations -> Compare window and select 2 different stacks and ensure that the stacks are named accordingly.
- Perform a safe apply operation and ensure the title is new stack and original stack as titles
